src/axom/mir/examples/tutorial_simple/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+set( mir_example_dependencies
+     core
+     slic
+     mir
+   )
+
+# Break the different MIR cases into multiple files to speed up compilation.
+set(mir_tutorial_simple_sources
+    mir_tutorial_simple.cpp
+    runMIR_seq.cpp
+    runMIR_seq_tri.cpp
+    runMIR_seq_quad.cpp
+    runMIR_seq_hex.cpp
+    runMIR_omp.cpp
+    runMIR_omp_tri.cpp
+    runMIR_omp_quad.cpp
+    runMIR_omp_hex.cpp
+    runMIR_cuda.cpp
+    runMIR_cuda_tri.cpp
+    runMIR_cuda_quad.cpp
+    runMIR_cuda_hex.cpp
+    runMIR_hip.cpp
+    runMIR_hip_tri.cpp
+    runMIR_hip_quad.cpp
+    runMIR_hip_hex.cpp
+    )
+
+axom_add_executable(
```

Comment:
Should we add some tests for these MIR examples?

Reply:
I added some tests that run the programs with various policies and look for text output that indicated they ran.